classes/AppKernel/Arr/Explorer.php

```diff
@@ -381,7 +381,7 @@ public function getInstanceData($instance)
     if ( empty($instance) )
       throw new Exception("Empty instance identifier");
     else if ( ! is_numeric($instance) )
-      throw new Exception("Malformed instance identifier '$instanceInfo'");
+      throw new Exception("Malformed instance identifier '$instance'");
```

Comment:
Not a good idea to log known bad data (that is presumably from an untrusted source). (Thinking primarily of log4j type vuln).

```suggestion
      throw new Exception("Malformed instance identifier");
```